cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -932,7 +946,9 @@ def _add_cfn_volume(self, id: str, shared_ebs: SharedEbs):
         return ec2.CfnVolume(
             scope=self,
             id=id,
-            availability_zone=self._cluster_config.head_node.networking.availability_zone,
+            availability_zone=self._cluster_config.head_node.networking.availability_zone
+            if self._cluster_config.head_node.networking.availability_zone
+            else "",
```

Comment:
Why are we passing an empty string?

Reply:
After our discussion, we mocked the return value of "pcluster.models.cluster_config.HeadNodeNetworking.availability_zone" to avoid getting None for the test